ansible/roles/azure-cloud-storage/tasks/delete-using-azcopy.yml

```diff
@@ -1,6 +1,16 @@
 ---
+- name: generate SAS token for azcopy
+  shell: |
+   sas_expiry=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
```

Comment:
If some azcopy operation takes more than 30m, it would fail right? Would 1 hour be a safe default?

Reply:
Updated to 1 hour.